worlds/hades/Items.py

```diff
@@ -0,0 +1,396 @@
+    "GunHiddenAspect": ItemData(hades_base_item_id+79, True)
+}
+
+item_table_traps : Dict[str, ItemData] ={
+    "MoneyPunishment" : ItemData(hades_base_item_id+80, False, False, True),
+    "HealthPunishment": ItemData(hades_base_item_id+81, False, False, True),
+}
+
+item_table_helpers : Dict[str, ItemData] ={
+    "MaxHealthHelper" : ItemData(hades_base_item_id+82, False, False, False),
+    "BoonBoostHelper" : ItemData(hades_base_item_id+83, False, False, False),
+    "InitialMoneyHelper" : ItemData(hades_base_item_id+84, False, False, False),
+}
+
+def create_filler_pool_options(options):
+    item_filler_options = []
+    if options.darkness_pack_value.value:
+        item_filler_options.append("Darkness")
+    if options.keys_pack_value.value:
+        item_filler_options.append("Keys")
+    if options.gemstones_pack_value.value:
+        item_filler_options.append("Gemstones")
+    if options.diamonds_pack_value.value:
+        item_filler_options.append("Diamonds")
+    if options.titan_blood_pack_value.value:
+        item_filler_options.append("TitanBlood")
+    if options.nectar_pack_value.value:
+        item_filler_options.append("Nectar")
+    if options.ambrosia_pack_value.value:
+        item_filler_options.append("Ambrosia")
+    if not item_filler_options:
+        item_filler_options.append("Darkness")
+    return item_filler_options
+
+def create_trap_pool():
+    return [trap for trap in item_table_traps.keys()]
+
+
+def create_pact_pool_amount(options) -> Dict[str, int]:
+    item_pool_pacts = {
+        "HardLaborPactLevel": int(options.hard_labor_pact_amount.value),
+        "LastingConsequencesPactLevel": int(options.lasting_consequences_pact_amount.value),
+        "ConvenienceFeePactLevel": int(options.convenience_fee_pact_amount.value),
+        "JurySummonsPactLevel": int(options.jury_summons_pact_amount.value),
+        "ExtremeMeasuresPactLevel": int(options.extreme_measures_pact_amount.value),
+        "CalisthenicsProgramPactLevel": int(options.calisthenics_program_pact_amount.value),
+        "BenefitsPackagePactLevel": int(options.benefits_package_pact_amount.value),
+        "MiddleManagementPactLevel": int(options.middle_management_pact_amount.value),
+        "UnderworldCustomsPactLevel": int(options.underworld_customs_pact_amount.value),
+        "ForcedOvertimePactLevel": int(options.forced_overtime_pact_amount.value),
+        "HeightenedSecurityPactLevel": int(options.heightened_security_pact_amount.value),
+        "RoutineInspectionPactLevel": int(options.routine_inspection_pact_amount.value),
+        "DamageControlPactLevel": int(options.damage_control_pact_amount.value),
+        "ApprovalProcessPactLevel": int(options.approval_process_pact_amount.value),
+        "TightDeadlinePactLevel": int(options.tight_deadline_pact_amount.value),
+        "PersonalLiabilityPactLevel": int(options.personal_liability_pact_amount.value),
+    }
+    return item_pool_pacts
```

Comment:
I'm pretty sure if you do .value, then these are all ints already? I'm not 100% sure on that, though

Reply:
yes that works